src/constants/TokenLogoLookupTable.ts

```diff
@@ -0,0 +1,37 @@
+import store from 'state'
+
+import { DEFAULT_LIST_OF_LISTS } from './lists'
+
+class TokenLogoLookupTable {
+  dict: { [key: string]: string[] | undefined } | null = null
```

Comment:
Why can't this be initialized in the constructor? Must it be initialized after the store has been initialized (in React)?

Reply:
I used a similar setup for TokenSafetyLookupTable.ts, & at the time connor wanted me to lazily create the map for that to decrease page load time if the lists potentially got larger.